Pipeline/SimplePackageDeploy/SimplePackageDeploy.properties

```diff
@@ -0,0 +1,12 @@
+########################################################################
+### The following properties are to configure the untar and deploy
+########################################################################
+
+# This copyModeMap JSON string consists of "deploy type": "the DBB CopyToFS CopyMode" pairs.
+# DBB supported copy modes are documented at
+# https://www.ibm.com/docs/api/v1/content/SS6T76_1.1.0/javadoc/com/ibm/dbb/build/DBBConstants.CopyMode.html
+copyModeMap = {"JCL": "TEXT", "DBRM": "BINARY", "LOAD": "LOAD", "CICSLOAD": "LOAD", "MAPLOAD": "LOAD", "IMSLOAD": "LOAD"}
```

Comment:
Something which the package actually should contain. Or is this a bad idea?

Reply:
As discussed, maintaining this as of now.  Will update once the BuildReport.json from PackageBuildOutputs is updated with copy type.